### PR TITLE
artweek6

### PR DESCRIPTION
**Pull request by art-ortega**
sorry about the delay!

### CODE REVIEW

File: code-challenges.js [hunk 1]
```diff
@@ -17,12 +32,28 @@ var testingArray1 = [23, "Heyyyy!", 45, -9, 0, "Yo", false]
...
 
+const rem = arr => {
+    let cool = []
+    arr.forEach(v => {
+        if(typeof v == 'number'){
+            cool.push(v%3)
+        }
+    })
+    return cool
+}
```
Comment (rachaelstanislaw) [inline code-challenges.js:43]: nice!

File: code-challenges.js [hunk 2]
```diff
@@ -17,12 +32,28 @@ var testingArray1 = [23, "Heyyyy!", 45, -9, 0, "Yo", false]
...
 // --------------------3) Create a function that takes in two arrays as arguments returns one array with no duplicate values.
 
 var testingArray3 = [3, 7, "hi", 10, 3, "hello", 4, "hi"]
 var testingArray4 = [7, "hi", 3, 1, "hi", 4, "hello", 4, 7]
 // Expected output: [ 3, 7, "hi", 10, "hello", 4, 1 ]
+
+const noDup = (arr1,arr2) =>{
+    let tight = arr1.concat(arr2)
+    return  tight.filter((v,i) => tight.indexOf(v) == i)
+}
```
Comment (rachaelstanislaw) [inline code-challenges.js:58]: excellent!

File: interview-questions.md [hunk 3]
```diff
@@ -3,39 +3,42 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. In a model called Animal that has_many Sightings, what is the name of the foreign key? Would the foreign key be part of the Animal model or the Sightings model?
 
-  Your answer:
+  Your answer: animal_id is the name of the foreign key. The foreign key would belong to the sightings model
```
Comment (rachaelstanislaw) [inline interview-questions.md:6]: yup!

File: interview-questions.md [hunk 4]
```diff
@@ -3,39 +3,42 @@ Answer the following questions. First, without external resources. Challenge you
...
+  Your answer: animal_id is the name of the foreign key. The foreign key would belong to the sightings model 
 
   Researched answer:
 
 
 
 2. Which routes must always be passed params and why?
 
-  Your answer:
+  Your answer: update, delete, patch because it needs to know which object its modifying
```
Comment (rachaelstanislaw) [inline interview-questions.md:14]: si!

File: interview-questions.md [hunk 5]
```diff
@@ -3,39 +3,42 @@ Answer the following questions. First, without external resources. Challenge you
...
+  Your answer: update, delete, patch because it needs to know which object its modifying 
 
   Researched answer:
 
 
 
 3. Write a rails route for a controller called "main" and a page called "game" that takes in a parameter called "guess".
 
-  Your answer:
+  Your answer:  get '/home/:guess' => 'main#game'
```
Comment (rachaelstanislaw) [inline interview-questions.md:22]: woohoo!

File: interview-questions.md [hunk 6]
```diff
@@ -3,39 +3,42 @@ Answer the following questions. First, without external resources. Challenge you
...
 
 
 
 4. Name three rails generator commands. What is created by each?
 
-  Your answer:
+  Your answer: model, resource, controller. The model generates a table with the attributes and data types for them. The controller generates a controller, and your views. Resource generates your model, controller, and sets up your routes for you.
```
Comment (rachaelstanislaw) [inline interview-questions.md:28]: mhmm!

File: interview-questions.md [hunk 7]
```diff
@@ -3,39 +3,42 @@ Answer the following questions. First, without external resources. Challenge you
...
 
 
 5. Consider the Rails routes below. Describe the responsibility of each route.
 
 /users        method="GET"   
 
+retrieves the information from the users path
+
 /users/1      method="GET"   
+retrieves the information
```
Comment (rachaelstanislaw) [inline interview-questions.md:41]: missed answering a couple of these :/

File: rails-commenting.rb [hunk 8]
```diff
@@ -36,15 +36,15 @@ def delete
...
       redirect_to blog_post_path(@post)
     end
   end
 
-  # 7)
+  # 7) makes 
   private
   def blog_post_params
-    # 8)
+    # 8) something, suggestion made by connor
```
Comment (rachaelstanislaw) [inline rails-commenting.rb:47]: hmm missed a couple answers here (num 7 - 10), but for the most part great!

**Review by rachaelstanislaw — APPROVED** [review]
let me know if you have questions then you're good to merge :)

File: rails-commenting.rb (hunk 8 above)
Comment (art-ortega) [reply rails-commenting.rb:47]: .... oh man, I did not intend on uploading this! It was a joke I made with connor in class, I thought I had edited it! .... sorry.

**Pull request merged**